### DISCUSSION TITLE
Consider removing generic protein (de)phosphorylation reactions

### DISCUSSION THREAD

**Discussion by JonathanRob** [SysBioChalmers/Human-GEM#336] [General]
### Note: This Discussion topic was resolved in #337.

Human-GEM currently contains a few generic protein phosphorylation/dephosphorylation reactions that are each associated with many genes:

Reaction ID | Equation | # Genes
:-- | :-- | :--
[MAR09577](https://metabolicatlas.org/explore/Human-GEM/gem-browser/reaction/MAR09577) | ATP [c] + [protein] [c] ⇒ ADP [c] + phosphoprotein [c] | 351
[MAR09578](https://metabolicatlas.org/explore/Human-GEM/gem-browser/reaction/MAR09578) | H2O [c] + phosphoprotein [c] ⇒ H+ [c] + Pi [c] + [protein] [c] | 78
[MAR09579](https://metabolicatlas.org/explore/Human-GEM/gem-browser/reaction/MAR09579) | ATP [c] + protamine [c] ⇒ ADP [c] + O-phosphoprotamine [c] | 212
[MAR07617](https://metabolicatlas.org/explore/Human-GEM/gem-browser/reaction/MAR07617) | ATP [c] + [protein]-L-tyrosine [c] ⇒ ADP [c] + H+ [c] + [protein]-tyrosine-phosphate [c] | 113
[MAR07618](https://metabolicatlas.org/explore/Human-GEM/gem-browser/reaction/MAR07618) | H2O [c] + [protein]-tyrosine-phosphate [c] ⇒ Pi [c] + [protein]-L-tyrosine [c] | 86

These reactions are all dead-ends, which is not a problem in itself, but they are associated with a very large number of primarily kinase-encoding genes. I am generally in support of trying to add more information to Human-GEM when possible, but it is important to keep in mind that as a _metabolic_ model, it is often necessary to limit the scope of the GEM to _metabolic_ processes. The definition of what is "metabolic" is relatively imprecise and will vary depending on who you ask, though I would argue that these hundreds of kinase-encoding genes are not so "metabolic" and fall more into the "signaling" category. They also do not contribute much information to the model, as they are all attached in a giant group to a single type of chemical transformation (addition/removal of a phosphate group).

I suggest that we remove these 5 reactions from Human-GEM, to avoid having so many generic kinases that do not participate anywhere else in the metabolic network. @johan-gson has also recommended this solution in the past.

What do others think?

**Comment by johan-gson** [comment]
I'm am in favor of this. It is clear to me that the GPRs are meaningless. What I don't fully understand is if the reactions are needed for something. But if they are dead-end, maybe they are not.

**Reply by haowang-bioinfo** [reply]
agree, inclusion of over generic reactions involving a lot of genes is not much useful to a GEM. Sometimes, this may give misleading results.

**Reply by mihai-sysbio** [reply]
How about mentioning this in the Gitter channel and then creating an issue for this?
